proto/decentraland/kernel/apis/scene.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package decentraland.kernel.apis;
+
+message GetSceneRequest {}
+message GetSceneResponse {
+    string cid = 1;
+    string scene_id = 2;
+    string scene_json = 3;
+}
```

Comment:
@gonpombo8 would you also add the baseUrl and the content mappings? These are part of every scene, available in the LoadableScene type of the explorer

https://github.com/decentraland/unity-renderer/blob/495c203d9b9e0c02cc6e4376deb92dc45ac7f54d/browser-interface/packages/shared/types.ts#L186